File: 69. Sqrt(x)/solution_1.cpp
class Solution {
    public:
        int mySqrt(int x) 
        {
            long int sqrt = 1;

            for (long int i = 0; i <= x; ++i)
            {
                if (i * i <= x)
                    sqrt = i;
                else
                    break;
            }

            return sqrt;
        }
};

File: 69. Sqrt(x)/solution_1.java
class Solution
{
    public int mySqrt(int x)
    {
        long sqrt = 0;

        for (long i = 0; i <= x; ++i)
        {
            if (i * i <= x)
                sqrt = i;
            else 
                break;
        }
        
        return (int)sqrt;
    }
}